tiatoolbox/tools/registration/wsi_registration.py

```diff
@@ -0,0 +1,32 @@
+import numpy as np
+from skimage import exposure, filters, morphology
+
+
+def preprocess(fixed_img, moving_img):
+    """Performs histogram normalization between two images.
+
+    Args:
+        fixed_img (:class:`numpy.ndarray`):
+            A grayscale fixed image
+        moving_img (:class:`numpy.ndarray`):
+            A grayscale moving image
+
+    Returns:
+        :class:`numpy.ndarray`:
+            A normalized grayscale fixed image
+        :class:`numpy.ndarray`:
+            A normalized grayscale moving image
+
+    """
+    if len(fixed_img.shape) != 2 or len(moving_img.shape) != 2:
+        raise ValueError("The input images should be grayscale images.")
+
+    moving_entropy, fixed_entropy = filters.rank.entropy(
+        moving_img, morphology.disk(3)
+    ), filters.rank.entropy(fixed_img, morphology.disk(3))
```

Comment:
Can you pass the size of the disk as a variable with default value =3? That would make it flexible for future extension.